src/site/apt/toolchains/custom.apt

```diff
@@ -25,22 +25,22 @@
 
 Custom Toolchains
 
-  You can create your own custom toolchains with plugins using them.
+  You can create your own custom toolchains for plugins to use.
```

Comment:
in fact, you need in 2 steps:
1. create a toolchain
2. have a plugin use it

Reply:
agree with @hboutemy - old sentence was more accurate for me also

Reply:
@elharo I'm thinking about release a new version of this plugin. Do you want to remove that line and merge the PR ?

Reply:
maybe `You can create your own custom toolchains to use with standard plugins (such as maven-compiler-plugin) or custom plugins` ?

Reply:
done, though I'm not sure "custom plugin" is really a thing. At this level, aren't they all just plugins?